src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -25,7 +25,7 @@
       <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
       <OSGroup>$(InputOSGroup)</OSGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
```

Comment:
As we chatted offline lets try to update the ProjectReference in System.Collections to get the netstandard1.5 version instead of this change.